apps/web/src/components/notebook/index.js

```diff
@@ -154,6 +155,18 @@ const menuItems = [
     iconColor: "error",
     icon: Icon.Trash,
     onClick: async ({ items }) => {
+      if (await showDeleteNotesDialog("Notebook")) {
+        items.map(async (item) => {
```

Comment:
`Array.map` is used for mapping each element in an array to a different value. Since there is no mapping being done here, why use `map` instead of a normal `for..of` loop?